junction/proposals/admin.py

```diff
@@ -15,7 +15,8 @@
     ProposalSection,
     ProposalSectionReviewer,
     ProposalType,
-    ProposalVote
+    ProposalVote,
+    ProposalSectionReviewerVote
```

Comment:
I think better to add one flag in ProposalVote models called reviewers or private.(I prefer reviewers to avoid confusion with comment section)